test/junit/scala/math/BigDecimalTest.scala

```diff
@@ -287,4 +287,29 @@ class BigDecimalTest {
   def testIsComparable(): Unit = {
     assert(BigDecimal(0.1).isInstanceOf[java.lang.Comparable[_]])
   }
+
+  
+
+  @Test
+  def testBigDecimalSumInList(): Unit = {
+
+    val bds = List(
+      scala.math.BigDecimal("1000000000000000000000000.1", MC.UNLIMITED),
+      scala.math.BigDecimal("9.0000000000000000000000009", MC.UNLIMITED))
+    assert(bds.sum == scala.math.BigDecimal("1000000000000000000000009.1000000000000000000000009", MC.UNLIMITED))
+
+    println("BigDecimal works")
+  }
+
+  @Test
+  def testBigDecimalProductList(): Unit = {
+    val bds = List(
+      scala.math.BigDecimal("1000000000000000000000000.1", MC.UNLIMITED),
+      scala.math.BigDecimal("9.00000000000000000000000091", MC.UNLIMITED))
+
+    val prod = bds.foldLeft(BigDecimal(1, MC.UNLIMITED))(_ * _)
+    assert(prod.toString == "9000000000000000000000001.810000000000000000000000091")
```

Comment:
it might be good to compare this to a `BigDecimal` rather than a `String`

```suggestion
    assert(prod == BigDecimal("9000000000000000000000001.810000000000000000000000091", MC.UNLIMITED))
```